pinot-core/src/main/java/org/apache/pinot/core/data/table/TableResizer.java

```diff
@@ -407,13 +420,25 @@ public Comparable extract(Record record) {
    */
   private class AggregationFunctionExtractor implements OrderByValueExtractor {
     final int _index;
+    final boolean _isFilteredAgg;
     final AggregationFunction _aggregationFunction;
 
     AggregationFunctionExtractor(int aggregationFunctionIndex) {
       _index = aggregationFunctionIndex + _numGroupByExpressions;
+      _isFilteredAgg = false;
       _aggregationFunction = _aggregationFunctions[aggregationFunctionIndex];
     }
 
+    AggregationFunctionExtractor(int aggregationFunctionIndex, boolean isFilteredAgg) {
```

Comment:
Suggest changing the constructor to take both `aggregationFunctionIndex` and `aggregationFunction` which is IMO easier to understand